src/darcyai/reporter/analytics_reporter.py

```diff
@@ -0,0 +1,416 @@
+import analytics
```

Comment:
Change package/module/file from `src/darcyai/reporter/analytics_reporter.py` to `src/darcyai/telemetry/telemetry_reporter.py`. Basically, `telemetry` is now the preferred name for `analytics`.